webpack.config.js

```diff
@@ -24,7 +25,7 @@ let CSS_PATH_STRING
 
 let { EDITOR_ORIGIN } = process.env
 let { EVAL_FRAME_ORIGIN } = process.env
-let { IODIDE_PUBLIC_TOS } = process.env || false
+const { IODIDE_PUBLIC_TOS } = process.env || false
```

Comment:
Please revert this, it's not related to the PR

Reply:
This is a linting issue  what I suppose to do?

Reply:
The linter in circle ci doesn't seem to be complaining about this line of code, so reverting it should not result in any errors on the PR.

In fact this probably *should* be const (same with the missing comma below), but we try to keep PRs focused on particular issues, and not an amalgamation of other changes. If you like, you can open up a follow-up to figure out why the linter isn't complaing about these lines in ci.